process_manager/templates/process_manager/partials/process_table.html

```diff
@@ -1,4 +1,4 @@
 {% load render_table from django_tables2 %}
-<div hx-post="{% url 'process_manager:process_table' %}"
-     hx-trigger="load delay:1s, click from:input"
+<div hx-get="{% url 'process_manager:process_table' %}"
```

Comment:
Is it `get` now because we are no longer sending data (the selected rows) to the backend?

Reply:
Right.